src/inverted-index.js

```diff
@@ -0,0 +1,134 @@
+/*  eslint-disable no-unused-vars*/
+/* jshint esnext: true */
+/**
+ * InvertedIndex class with constructor
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.file = {};
+    this.index = {};
+    this.docNumber = {};
+  }
+
+  /**
+   *
+   *
+   * @param {any} fileContent
+   * @param {any} fileName
+   * @returns {string} returns createIndex
+   * @memberOf InvertedIndex
+   */
+  createIndex(fileContent, fileName) {
+    this.docCount = [];
+    fileContent.forEach((bookContent, docIndex) => {
+      this.docCount.push(parseInt(docIndex, 10));
+      this.combinedWord = (`${bookContent.title} ${bookContent.text}`).toLowerCase();
+      const token = InvertedIndex.tokenize(this.combinedWord);
```

Comment:
**Nitpick - Code Quality**
This is an array of words so `tokens` would be a more appropriate variable name.